pyigm/guis/igmguesses.py

```diff
@@ -1421,13 +1435,13 @@ def __init__(self, component=None, parent=None):
         self.parent = parent
         #if not status is None:
         #    self.statusBar = status
-        self.label = QtGui.QLabel('Component:',self)
+        self.label = QLabel('Component:',self)
         self.zwidget = ltgsm.EditBox(-1., 'zc=', '{:0.5f}')
```

Comment:
It seems to me that we would need to create a button on the widget like "Update", for when the user is done editing the box(es) the values are retrieved and updated after pressing such button. What do you think?